solver/src/driver/solver_settlements.rs

```diff
@@ -257,4 +267,40 @@ mod tests {
         let settlements = vec![settlement.clone(), settlement];
         assert!(merge_at_most_settlements(1, settlements.into_iter()).is_none());
     }
+
+    #[test]
+    fn compute_objective_value() {
+        // Gas price is 2e-7 ETH
+        let gas_price = BigRational::from_integer(200_000_000_000_u128.into());
+
+        // Surplus1 is 0.4 ETH
+        let surplus1 = BigRational::from_integer(400_000_000_000_000_000_u128.into());
+
+        // Surplus2 is 1.0 ETH
+        let surplus2 = BigRational::from_integer(1_000_000_000_000_000_000_u128.into());
+
+        // Gas estimate1 is 500_000
+        let gas_estimate1 = BigRational::from_integer(500_000.into());
+
+        // Gas estimate2 is 100_000
+        let gas_estimate2 = BigRational::from_integer(100_000.into());
+
+        // Objective value 1 is 4e-1 - 5e5 * 2e-7 = 4e-1 - 1e-1 = 0.3 ETH
+        let obj_value1 =
+            RatedSettlement::compute_objective_value(&surplus1, &gas_estimate1, &gas_price);
+
+        assert_eq!(
```

Comment:
Why are there two asserts here? What is the difference between the two scenarios? Could their difference be documented by using using two descriptively named unit tests. I'd also still like to see the scenario where we have two solutions. One being ranked higher at a certain low gas price, maybe them being equal at some gas price and then the second one ending up higher ranked at a sufficiently high gas price.